### PR TITLE
Update MSBuild.sln for MSBuildTaskHost

### PR DESCRIPTION
**Pull request by jeffkl**
This change just updates the version of Visual Studio and maps AnyCPU for MSBuildTaskHost to x86 which is disabled anyway.

### CODE REVIEW

**Comment by jeffkl** [comment]
@dotnet-bot test Windows_NT Build for CoreCLR please

File: src/MSBuild.sln [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
-VisualStudioVersion = 15.0.26228.9
+VisualStudioVersion = 15.0.26730.0
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.Build.Framework", "Framework\Microsoft.Build.Framework.csproj", "{571F09DB-A81A-4444-945C-6F7B530054CD}"
```
Comment (themusicplace61) [inline src/MSBuild.sln]: [licenseKOR (2014_04_26 04_57_13 UTC).txt](https://github.com/Microsoft/msbuild/files/1278500/licenseKOR.2014_04_26.04_57_13.UTC.txt)
[Uploading hppsdrlog (2014_06_10 06_23_59 UTC).txt…]()

Comment (themusicplace61) [reply src/MSBuild.sln]: #

File: src/MSBuild.sln [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
-VisualStudioVersion = 15.0.26228.9
+VisualStudioVersion = 15.0.26730.0
```
Comment (themusicplace61) [inline src/MSBuild.sln]: "[json]": {}
#2489 

[licenseKOR (2014_04_26 04_57_13 UTC).txt](https://github.com/Microsoft/msbuild/files/1278566/licenseKOR.2014_04_26.04_57_13.UTC.txt)

**Pull request merged**